packages/fxa-auth-server/lib/senders/emails/global.scss

```diff
@@ -67,6 +67,9 @@ $s-10: 40px;
   &-8 {
     margin-top: $s-8 !important;
   }
+  &-10 {
+    margin-top: $s-10 !important;
+  }
```

Comment:
Kinda wonder if it would be worth putting these `$s-X` values into a map and then iteratively generating all the `.mX-Y` classes. This wouldn't result in excessive classes since they're inlined anyway.

Reply:
I agree, though I just pushed an update containing everything except this - maybe one of us can take a stab at it with the next set of templates. It might also be worth separating some of these out into separate scss files and then importing them into this global file.